Signal/src/ViewControllers/OversizeTextMessageViewController.swift

```diff
@@ -10,87 +10,24 @@ class OversizeTextMessageViewController: OWSViewController {
 
     let TAG = "[OversizeTextMessageViewController]"
 
-    let message: TSMessage
+    let displayableText: String
+    let attachmentStream: TSAttachmentStream
 
     // MARK: Initializers
 
     @available(*, unavailable, message:"use message: constructor instead.")
     required init?(coder aDecoder: NSCoder) {
-        message = TSMessage()
+        displayableText = ""
+        attachmentStream = TSAttachmentStream(contentType:"", sourceFilename:"")
         super.init(coder: aDecoder)
     }
 
-    required init(message: TSMessage) {
-        self.message = message
+    required init(displayableText: String, attachmentStream: TSAttachmentStream) {
+        self.displayableText = displayableText
+        self.attachmentStream = attachmentStream
```

Comment:
Here too we can dramatically simplify a class by passing in the state it needs from our view item.